src/containers/Homepage.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import Header from '../components/Header';
+import Sidebar from '../components/Sidebar.js';
+import BookingPage from '../components/BookingPage.js';
+import '../sass/Homepage.scss';
```

Comment:
usually to save the stylesheets we will be saving into the folder stylesheets and if the stylesheet is related with the containers or components we will be putting the scss inside the particular folder and import them here

Reply:
so can you rename the sass folder into stylesheets